### ISSUE TITLE
Decode error

### ISSUE THREAD

**Issue by brunsgaard** [CERN-CERT/WAD#14]
I found this bug, the packages are installed with `pip`.

```
(venv) ➜  python --version
Python 3.5.0
(venv) ➜  pip freeze
six==1.10.0
wad==0.3.4
(venv) ➜  wad -u http://designfund.dk
Traceback (most recent call last):
  File "/Users/brunsgaard/clerk/leadsland/venv/bin/wad", line 9, in <module>
    load_entry_point('wad==0.3.4', 'console_scripts', 'wad')()
  File "/Users/brunsgaard/clerk/leadsland/venv/lib/python3.5/site-packages/wad/__main__.py", line 94, in main
    results = Detector().detect_multiple(urls, limit=options.limit, exclude=options.exclude, timeout=timeout)
  File "/Users/brunsgaard/clerk/leadsland/venv/lib/python3.5/site-packages/wad/detection.py", line 78, in detect_multiple
    res = self.detect(url, limit, exclude, timeout)
  File "/Users/brunsgaard/clerk/leadsland/venv/lib/python3.5/site-packages/wad/detection.py", line 54, in detect
    content = content.decode()
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe6 in position 799: invalid continuation byte
```

**Comment by plizonczyk** [comment]
Thank you for the report and sorry for the delay - I'm looking into it.

**Comment by nikofil** [comment]
Has been resolved in #22
